client/src/entrypoints/admin/bulk-actions.js

```diff
@@ -16,6 +17,30 @@ const checkedState = {
   shouldShowAllInListingText: true,
 };
 
+/**
+ * Hides the 'more' dropdown button in listing pages.
+ */
+function hideMoreActionsDropdownBtn() {
+  const MoreActionsDropdown = document.querySelector(
```

Comment:
nitpick: `MoreActionsDropdown` -> `moreActionsDropdown`

```suggestion
  const moreActionsDropdown = document.querySelector(
```

my preference would have been to have a single `toggleMoreActionsDrodown` function that accepts a show/hide parameter, but this is fine too

Reply:
I've replaced the previous functions by a unique `toggleMoreActionsDropdownBtn` function.